### ISSUE TITLE
[Bug] Trash Cans

### ISSUE THREAD

**Issue by Pattycakebakersmaan** [Zonkeeh/ONI-Mods#74]
**Steam Name**
hypnotoad2016
https://steamcommunity.com/profiles/76561198343704206/

**Operating System / System requirements**
Windows 10

**Mod Version**
Your current version of the mod (found in output log)
Build: U43-526233-SD

**Game Version**
Sweet Dreams update

**DLC**
Spaced Out DLC

**Describe the bug**
Crashes as soon as trash can is built

**To Reproduce**
Steps to reproduce the behaviour:
1. Start game
2. Build trash can
3. Crash

**Expected Behaviour**
Trash can built and doesn't crash

**Output Log**
MissingMethodException: !4 StateMachine`4/Parameter`1.Set(!4,!1)

Trashcans.TrashcanAnim+States+<>c.<InitializeStates>b__5_0 (Trashcans.TrashcanAnim+SMInstance smi, System.Single dt) (at <7b68900a49c348c8962fe07d773c12e0>:0)
BucketUpdater`1[DataType].Update (DataType data, System.Single dt) (at <1b5c2ff4f53d4a48b7159ed84d905978>:0)
UpdateBucketWithUpdater`1[DataType].Update (System.Single dt) (at <1b5c2ff4f53d4a48b7159ed84d905978>:0)
StateMachineUpdater+BucketGroup.AdvanceOneSubTick (System.Single dt) (at <1b5c2ff4f53d4a48b7159ed84d905978>:0)
StateMachineUpdater.AdvanceOneSimSubTick () (at <1b5c2ff4f53d4a48b7159ed84d905978>:0)
Game.SimEveryTick (System.Single dt) (at <cb20703507b14eea8088dbc44abf7f85>:0)
Game.Update () (at <cb20703507b14eea8088dbc44abf7f85>:0)

**Comment by Quinn-John88** [comment]
Just doing a quick google search, I found that this may be solvable by rebuilding/redeploying the assemblies. [Here](https://stackoverflow.com/questions/8058832/system-missingmethodexception-method-not-found) is a link to the stack overflow related. Hopefully this helps!

**Comment by Pattycakebakersmaan** [comment]
Sorry I have no idea what any of that means. I'm just trying to get the attention of the mod author since I and others would love to see this updated. As of now it is completely broken.



> Just doing a quick google search, I found that this may be solvable by rebuilding/redeploying the assemblies. [Here](https://stackoverflow.com/questions/8058832/system-missingmethodexception-method-not-found) is a link to the stack overflow related. Hopefully this helps!

**Comment by Quinn-John88** [comment]
@Pattycakebakersmaan Yeah, my reply was more so for the mod author. I should've been more explicit with my intention. That's my bad.
